src/features/HashlessTileCoder.jl

```diff
@@ -0,0 +1,145 @@
+
+export HashlessTileCoder, HashlessTileCoderFull
```

Comment:
You only need this once in the main file.

Reply:
```suggestion

```